src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -52,6 +52,9 @@ ucs_config_field_t uct_rc_mlx5_iface_config_table[] = {
 };
 
 
+/* Max value for log_ack_req_freq field in QPC */
+#define UCT_RC_MLX5_MAX_LOG_ACK_REQ_FREQ (8)
```

Comment:
1. pls move defines to beginning of C file
2. can we use it in config option documentation string?

Reply:
1 - done.
2 - IMO no, because it is an int token, while the docstring requires a string.

Reply:
2- how about macro token stringification , or use UCS_PP_MAKE_STRING helper macro?

Reply:
Done